src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -2211,6 +2212,14 @@ public async Task Http2_MultipleConnectionsEnabled_OpenAndCloseMultipleConnectio
             }
         }
 
+        public static IEnumerable<object[]> ManyIterations()
+        {
+            for (var i = 0; i < 1000; i++)
+            {
+                yield return new object[] {i};
+            }
+        }
```

Comment:
This is just for testing CI, right?  You'll remove this and put it back to being a fact before merging?

Reply:
Yes, that's correct.